R/bs-theme-preview.R

```diff
@@ -389,13 +389,13 @@ bs_get_variables <- function(theme, varnames) {
     collapse = "\n"
   )
 
-  # Declare a block with a meaningless but identifiable selector (:root.get_default_vars)
+  # Declare a block with a meaningless but identifiable selector (.get_default_vars:root)
   # and add properties for each variable that is desired.
   cssvars <- paste0(
     "--", varnames, ": #{inspect($", varnames, ")};",
     collapse = "\n"
   )
-  cssvars <- sprintf(":root.get_default_vars {\n %s \n}", cssvars)
+  cssvars <- sprintf(".get_default_vars:root {\n %s \n}", cssvars)
```

Comment:
Why is this needed?

Reply:
When the newly updated Sass compiles it, it was re-ordered.